src/commands.ts

```diff
@@ -223,8 +223,14 @@ export const inputHandlers = (store: Store<State, any>) => ({
 
     // Get the command from the command gesture index.
     // When the command palette  is displayed, disable gesture aliases (i.e. gestures hidden from instructions). This is because the gesture hints are meant only as an aid when entering gestures quickly.
-    const command =
-      !state.showCommandPalette || !commandGestureIndex[sequence as string]?.hideFromHelp
+    const helpCommand = commandIdIndex['help']
+    const helpGesture = helpCommand?.gesture as string
```

Comment:
You can use the `gestureString` helper to get the normalized gesture string:

https://github.com/cybersemics/em/blob/75b8c516e70922bdfa44d29a8cc47e33c942e865/src/commands.ts#L321-L323